cmd/populate_user_table/sql/get_all_usernames.sql

```diff
@@ -7,179 +7,180 @@ SELECT username
+    SELECT created_by AS username
+    FROM trb_admin_notes_trb_admin_note_recommendations_links
+    UNION
+    SELECT modified_by AS username
+    FROM trb_admin_notes_trb_admin_note_recommendations_links
+
+    UNION
+
+    -- Sub query for trb_admin_notes_trb_request_documents_links table
+    SELECT created_by AS username
+    FROM trb_admin_notes_trb_request_documents_links
+    UNION
+    SELECT modified_by AS username
+    FROM trb_admin_notes_trb_request_documents_links
+
+    UNION
+
+    -- Sub query for trb_guidance_letter_recommendations table
+    SELECT created_by AS username
+    FROM trb_guidance_letter_recommendations
+    UNION
+    SELECT modified_by AS username
+    FROM trb_guidance_letter_recommendations
+
+    UNION
+
+    -- Sub query for trb_guidance_letters table
+    SELECT created_by AS username
+    FROM trb_guidance_letters
+    UNION
+    SELECT modified_by AS username
+    FROM trb_guidance_letters
+
+    UNION
+
+    -- Sub query for trb_lead_options table
+    SELECT created_by AS username
+    FROM trb_lead_options
+    UNION
+    SELECT modified_by AS username
+    FROM trb_lead_options
+    UNION
+    SELECT eua_user_id AS username
+    FROM trb_lead_options
+
+    UNION
+
+    -- Sub query for trb_request table
+    SELECT created_by AS username
+    FROM trb_request
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request
+    UNION
+    SELECT trb_lead AS username
+    FROM trb_request
+
+    UNION
+
+    -- Sub query for trb_request_attendees table
+    SELECT created_by AS username
+    FROM trb_request_attendees
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request_attendees
+    UNION
+    SELECT eua_user_id AS username
+    FROM trb_request_attendees
+
+    UNION
+
+    -- Sub query for trb_request_documents table
+    SELECT created_by AS username
+    FROM trb_request_documents
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request_documents
+
+    UNION
+
+    -- Sub query for trb_request_feedback table
+    SELECT unnest(notify_eua_ids) AS username
+    FROM trb_request_feedback
+    UNION
+    SELECT created_by AS username
+    FROM trb_request_feedback
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request_feedback
+
+    UNION
+
+    -- Sub query for trb_request_forms table
+    SELECT created_by AS username
+    FROM trb_request_forms
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request_forms
+
+    UNION
+
+    -- Sub query for trb_request_funding_sources table
+    SELECT created_by AS username
+    FROM trb_request_funding_sources
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request_funding_sources
+
+    UNION
+
+    -- Sub query for trb_request_system_intakes table
+    SELECT created_by AS username
+    FROM trb_request_system_intakes
+    UNION
+    SELECT modified_by AS username
+    FROM trb_request_system_intakes
+) AS combined_usernames
```

Comment:
Side note: good lord what is this? Unions pulling data from every table?

Reply:
@mynar7 , yeah this was for migrating to the user account table. We needed to get the usernames that might not have a user account representation.